test/models/download_request_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+class DownloadRequestTest < ActiveSupport::TestCase
+
+  before(:all) do
```

Comment:
no need to have this is a before(:all) block since it is only used in one test, and does not save anything to the DB.  Might as well keep the tests self-contained and move this setup to inside the test itself